setup.py

```diff
@@ -23,6 +23,7 @@
     'pyopenssl',
     'requests>=2.13.0',
     'daemon',
+    'defusedxml',
```

Comment:
Can the reason for this be explained?

Reply:
It was added a few bugfixes ago by an outside contributor. I forget which module, but there were security concerns with the xml processing and he was tired of patching his copy of hubble, so he submitted a PR that fixed various things about xml processing in that module. I objected to defused xml at first, but after looking at it... it's worth using for sure.